addons/web/static/tests/core/l10n/translation_tests.js

```diff
@@ -100,6 +100,39 @@ QUnit.test("luxon is configured in the correct lang", async (assert) => {
     assert.strictEqual(DateTime.utc(2021, 12, 10).toFormat("MMMM"), "décembre");
 });
 
+QUnit.test("lang is given by an attribute on the DOM root node", async (assert) => {
+    patchWithCleanup(session.user_context, { lang: null });
+    document.documentElement.setAttribute("lang", "fr-FR");
+    registerCleanup(() => {
+        document.documentElement.removeAttribute("lang");
+    });
+    patchWithCleanup(session, {
+        cache_hashes: { translations: 1 },
+    })
+    patchWithCleanup(browser, {
+        fetch(url) {
+            assert.strictEqual(url, "/web/webclient/translations/1?lang=fr_FR");
```

Comment:
You should add a `assert.step` here or add `assert.expect(?)` at the beginning of the test.
Without this, we could pass through the function n times without knowing it.